[gftools-qa] error with browser previews

Found in https://github.com/google/fonts/pull/2922/checks?check_run_id=1776684330
Font doesn't have a weight axis.

```
INFO:__main__:Running browser previews
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.6/x64/bin/gftools-qa.py", line 502, in <module>
    main()
  File "/opt/hostedtoolcache/Python/3.8.6/x64/bin/gftools-qa.py", line 481, in main
    qa.googlefonts_new()
  File "/opt/hostedtoolcache/Python/3.8.6/x64/bin/gftools-qa.py", line 261, in googlefonts_new
    self.browser_previews()
  File "/opt/hostedtoolcache/Python/3.8.6/x64/bin/gftools-qa.py", line 245, in browser_previews
    html = HtmlProof(
  File "/opt/hostedtoolcache/Python/3.8.6/x64/lib/python3.8/site-packages/gftools/html.py", line 492, in __init__
    self.css_font_faces = css_font_faces(self.ttFonts, self.out)
  File "/opt/hostedtoolcache/Python/3.8.6/x64/lib/python3.8/site-packages/gftools/html.py", line 209, in css_font_faces
    font_weight=font_weight,
UnboundLocalError: local variable 'font_weight' referenced before assignment
Error: Process completed with exit code 1.
```

--- a/Lib/gftools/html.py
+++ b/Lib/gftools/html.py
@@ -181,6 +181,7 @@ def css_font_faces(ttFonts, server_dir=None, position=None):
         src = f"url({path})"
         font_family = _class_name(family_name, style_name, position)
         font_style = "italic" if font_is_italic(ttFont) else "normal"
+        font_weight = css_font_weight(ttFont)
         font_stretch = WIDTH_CLASS_TO_CSS[ttFont["OS/2"].usWidthClass]
 
         if "fvar" in ttFont:
@@ -200,8 +201,7 @@ def css_font_faces(ttFonts, server_dir=None, position=None):
                 min_angle = int(axes["slnt"].minValue)
                 max_angle = int(axes["slnt"].maxValue)
                 font_style = f"oblique {min_angle}deg {max_angle}deg"
-        else:
-            font_weight = css_font_weight(ttFont)
+
         font_face = CSSElement(
             "@font-face",
             src=src,
@@ -279,7 +279,11 @@ def css_font_classes_from_vf(ttFont, position=None):
 
         class_name = _class_name(family_name, inst_style, position)
         font_family = _class_name(family_name, style_name, position)
-        font_weight = int(instance.coordinates["wght"])
+        font_weight = (
+            css_font_weight(ttFont)
+            if not "wght" in instance.coordinates
+            else int(instance.coordinates["wght"])
+        )
         font_style = "italic" if "Italic" in inst_style else "normal"
         font_stretch = (
             "100%"

